compiler/rustc_codegen_llvm/src/intrinsic.rs

```diff
@@ -498,6 +498,23 @@ impl<'ll, 'tcx> IntrinsicCallBuilderMethods<'tcx> for Builder<'_, 'll, 'tcx> {
                 }
             }
 
+            sym::untyped_swap_nonoverlapping => {
```

Comment:
Is there a good reason for why this is handled in a different way than `typed_swap_nonoverlapping`, and in a completely different place as well?